tasks/preflight.yml

```diff
@@ -75,3 +75,10 @@
   with_items:
     - "{{ lookup('url', 'https://github.com/prometheus/prometheus/releases/download/v' + prometheus_version + '/sha256sums.txt', wantlist=True) | list }}"
   when: "('linux-' + (go_arch_map[ansible_architecture] | default(ansible_architecture)) + '.tar.gz') in item"
+
+- name: Get systemd version
+  shell: "systemctl --version | head -n1 | sed 's/systemd //g'"
```

Comment:
This might be less fragile:

```
systemd --version | awk '$1 == "systemd" {print $2}'
```

Reply:
I cannot use `systemd` command in test suite, since it is not detected in ubuntu and debian containers, but `systemctl` works fine and it is always installed with systemd. As for `awk` it is indeed better.

Reply:
Sorry, coffee is not kicked in yet.  That was supposed to be the same `systemctl` command.

Reply:
No problem :smile:

Ok, do you think there is anyting more to do in this PR?